pkgs/by-name/wo/workout-tracker/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  buildPackages,
+}:
+
+buildGoModule rec {
+  pname = "workout-tracker";
+  version = "0.10.4";
+
+  src = fetchFromGitHub {
+    owner = "jovandeginste";
+    repo = "workout-tracker";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-vMUNc5sBM02UvevETnDUqgDrnSifmIK+MCTJOgotjrE=";
+  };
+
+  vendorHash = null;
+
+  doCheck = false;
+
+  proxyVendor = true;
```

Comment:
Why is that required?

Reply:
removed. `vendorHash = null;` is required because upstream includes their own `vendor` dir.